src/libexpr/primops.cc

```diff
@@ -4074,54 +4093,210 @@ void EvalState::createBaseEnv()
 
     /* `builtins' must be first! */
     v.mkAttrs(buildBindings(128).finish());
-    addConstant("builtins", v);
+    addConstant("builtins", v, {
+        .type = nAttrs,
+        .doc = R"(
+          Contains all the [built-in functions](./builtins.md) and values, in order to avoid polluting the global scope.
+
+          Since built-in functions were added over time, [testing for attributes](./operators.md#has-attribute) in `builtins` can be used for graceful fallback on older Nix installations:
+
+          ```nix
+          if builtins ? getEnv then builtins.getEnv "PATH" else ""
+          ```
```

Comment:
This is not up to date with the current manual.

Reply:
What is different? When I do `git show` it looks like it matches the deleted `doc/manual/src/language/builtin-constants.md` contents.

Reply:
We discussed in chat, this was read herring it is up to date.